crmsh/ra.py

```diff
@@ -428,9 +429,10 @@ def actions(self):
             name = elem.get("name")
             if not name or name in self.skip_ops:
                 continue
-            d = {}
+            d = OrderedDict()
+            d['interval'] = elem.get('interval') if 'interval' in elem.attrib else '0s'
             for key in list(elem.attrib.keys()):
-                if key in self.skip_op_attr:
+                if key in self.skip_op_attr or d == 'interval':
```

Comment:
Is it `key == 'interval'`?

Can this chunk be simplified into setting `d['interval']` to `0s` and overwrite it in the loop?

Reply:
Changed